src/setup/setup.spec.ts

```
@@ -235,9 +235,8 @@ describe('Talawa Admin Setup', () => {
   };
 
   it('should test the validation function for reCAPTCHA site key with valid input', async () => {
-    const { validateRecaptcha } = await import(
-      './validateRecaptcha/validateRecaptcha'
-    );
+    const { validateRecaptcha } =
+      await import('./validateRecaptcha/validateRecaptcha');
```

ESLint

[error] Replace `⏎······await·import('./validateRecaptcha/validateRecaptcha'` with `·await·import(⏎······'./validateRecaptcha/validateRecaptcha'⏎····`

(prettier/prettier)